src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1834,6 +1835,10 @@ format layout_optimizer::get_preferred_format(program_node& node) {
         // Set default format for issue 92967/98750
         // TODO: will remove when arg_max_min_ref supports blocked format
         expected = format::get_default_format(node.get_input_layouts()[0].get_rank(), false, false);
+    } else if (node.is_type<scatter_nd_update>()) {
+        // 2nd, 3rd input should not be reordered
+        node.set_preferred_input_fmt(1, get_preferred_format(node.get_dependency(1)));
+        node.set_preferred_input_fmt(2, get_preferred_format(node.get_dependency(2)));
```

Comment:
I think we should do it not only for scatter_nd_update but also for all primitive in new_shape_infer scenario.

Reply:
done